users/models.py

```diff
@@ -0,0 +1,54 @@
+"""
+Holds database models
+"""
+from django.contrib.auth.models import AbstractBaseUser, PermissionsMixin
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from .managers import UserManager
+
+# pylint: disable = no-member
+
+
+class TimeStamp(models.Model):
+    """
+    TimeStamp model to check when the object was created and modified
+    """
+
+    created_on = models.DateTimeField(auto_now_add=True, editable=False)
+    updated_on = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        """
+        Tells that this class is abstract
+        """
+
+        abstract = True
+
+
+class User(AbstractBaseUser, PermissionsMixin, TimeStamp):
+    """
+    Customized user model for E-commerce app
+    """
+
+    email = models.EmailField(_("email address"), unique=True, editable=False)
```

Comment:
This should be an editable field in case we want to change the email for a user.

Reply:
I have updated it.